pkg/webui/console/lib/device-utils.js

```diff
@@ -156,3 +156,7 @@ export const getLastSeen = device => {
     }
   }
 }
+
+export const isUsBand = (band = '') => band.startsWith('US_902_928')
+export const isAuBand = (band = '') => band.startsWith('AU_915_928')
+export const isCnBand = (band = '') => band.startsWith('CN_470_510')
```

Comment:
Please double-check @johanstokking. We will disallow the setting of factory preset settings for all such bands.